.gitignore

```diff
@@ -0,0 +1 @@
+/.idea/
```

Comment:
please update the gitignore, at less it should ignore the target

Reply:
it should be 
/target/
/.idea/
/*.iml

Reply:
I fixed it